web-client/integration-tests/journey/petitionsClerkEditOtherStatisticToCase.js

```diff
@@ -0,0 +1,18 @@
+export const petitionsClerkEditOtherStatisticToCase = test => {
+  return it('petitions clerk edits other statistic on the case', async () => {
+    await test.runSequence('gotoEditOtherStatisticsSequence', {
+      docketNumber: test.docketNumber,
+    });
+    expect(test.getState('caseDetail.damages')).toEqual(5678);
```

Comment:
not sure why we're checking this when we don't update it later in the test @ericsorenson